ament_cmake_auto/ament_cmake_auto-extras.cmake

```diff
@@ -17,7 +17,9 @@
 find_package(ament_cmake QUIET REQUIRED)
 
 include("${ament_cmake_auto_DIR}/ament_auto_add_executable.cmake")
+include("${ament_cmake_auto_DIR}/ament_auto_add_gtest.cmake")
```

Comment:
@JWhitleyWork meta: this creates a dependency between `ament_cmake_auto` and `ament_cmake_gtest`, that's also not explicit. It could be made explicit, but have you considered splitting this into its own package?

Reply:
Explicit dependency added. I think that this dependency is fine. Gaining the "automatic" functionality in exchange for this dependency seems like a reasonable trade to me.